include/execution.hpp

```diff
@@ -2867,6 +3077,15 @@ namespace _P2300::execution {
         noexcept(nothrow_tag_invocable<ensure_started_t, _Sender>) {
         return tag_invoke(ensure_started_t{}, (_Sender&&) __sndr);
       }
+      template <sender _Sender>
+        requires (!__tag_invocable_with_completion_scheduler<ensure_started_t, set_value_t, _Sender>) &&
+          (!tag_invocable<ensure_started_t, _Sender>)
```

Comment:
```suggestion
        requires (!__tag_invocable_with_completion_scheduler<ensure_started_t, set_value_t, _Sender>) &&
          (!tag_invocable<ensure_started_t, _Sender>) &&
          sender<_Sender, make_env_t<with_t<get_stop_token_t, in_place_stop_token>>>
```

We repeat the type of the `ensure_started` environment in a number of places. Probably best to create an alias.